tests/manage/pv_services/test_resource_deletion_during_pvc_pod_deletion_and_io.py

```diff
@@ -99,6 +112,20 @@ def delete_pods(self, pods_to_delete):
             pod_obj.delete(wait=False)
         return True
 
+    def run_io_on_pods(self, pod_objs):
+        """
+        Run IO on pods
+        """
+        for pod_obj in pod_objs:
+            if pod_obj.pvc.access_mode == constants.ACCESS_MODE_RWX:
+                io_size = int((self.pvc_size - 1) / 2)
```

Comment:
why half?

Reply:
There will be two pods which use same RWX PVC. So distributing the size accordingly.